classes/AlertView.php

```diff
@@ -136,4 +162,47 @@ private function isEmailSignedUpForPostCode($email, $postcode) {
         }
         return $is_signed_up;
     }
+
+    private function getNewMP($confirmation) {
+        if (!$confirmation) {
+            return array();
+        }
+
+        $existing = $this->alert->fetch_by_token($confirmation);
+        $criteria = str_replace('speaker:', '', $existing['criteria']);
+        $data = array();
+
+        $old_mp = new Member(array( 'person_id' => $criteria ) );
+        if ( $old_mp->current_member_anywhere() ) {
+            $data = array(
+                'old_mp' => $old_mp->full_name(),
+                'still_in_post' => True,
```

Comment:
Not sure what `still_in_post` is for – is it needed, does the script not make sure this doesn't happen?